src/routes/(landing)/+layout.svelte

```
@@ -0,0 +1,77 @@
+        <!-- Navigation Links -->
+        <div class="hidden items-center gap-1 md:flex">
+          {#each navItems as item}
+            <Button
+              variant={currentPath === item.href ? "default" : "ghost"}
+              class="gap-2"
+              onclick={() => goto(item.href)}
+            >
+              <span>{item.icon}</span>
+              {item.label}
+            </Button>
+          {/each}
```

ESLint

[error] Each block should have a key

(svelte/require-each-key)